docs/Users_Guide/wrappers.rst

```diff
@@ -6875,8 +6875,147 @@ METplus Configuration
 | :term:`POINT2GRID_INC_VALID_TIMES`
 | :term:`POINT2GRID_SKIP_INIT_TIMES`
 | :term:`POINT2GRID_INC_INIT_TIMES`
+| :term:`POINT2GRID_VALID_TIME`
+| :term:`POINT2GRID_OBS_WINDOW_BEG`
+| :term:`POINT2GRID_OBS_WINDOW_END`
+| :term:`POINT2GRID_MESSAGE_TYPE`
+| :term:`POINT2GRID_VAR_NAME_MAP<n>_KEY`
+| :term:`POINT2GRID_VAR_NAME_MAP<n>_VAL`
+| :term:`POINT2GRID_OBS_QUALITY_INC`
+| :term:`POINT2GRID_OBS_QUALITY_EXC`
+| :term:`POINT2GRID_TIME_OFFSET_WARNING`
```

Comment:
Not sure if we should add `POINT2GRID_TIME_OFFSET_WARNING` since it isn't actually being used.